common/src/main/java/com/wynntils/models/items/ItemEncodingModel.java

```diff
@@ -13,6 +15,12 @@
 import java.util.regex.Pattern;
 
 public class ItemEncodingModel extends Model {
+    @Persisted
```

Comment:
This should have been configs, but we still cannot do configs in models, right? How do we change the value of these?

Reply:
Ah, I see, in the item sharing gui...